### PR TITLE
Bw new game

### PR DESCRIPTION
**Pull request by bmwwoo**

### CODE REVIEW

**Comment by bmwwoo** [comment]
@sf-interns/owners Once you guys let this PR in, we should merge it into gh-pages because goals, new game, and reset all work

File: app/controllers/MathSwipeController.coffee [hunk 1]
```diff
@@ -86,21 +96,10 @@ class MathSwipeController
...
-      sum = 0
-      for j in list
-        sum += j
-      if sum != size
-        console.log "Something went wrong with RandomizedFitLength"
-        console.log list
-        break
+    list = RandomizedFitLength.generate size
     console.log list
     console.log "Passed RandomizedFitLength"
```
Comment (shubhamgogna) [inline app/controllers/MathSwipeController.coffee:101]: I'm not sure this is even testing anymore, so can we remove it?

**Comment by shubhamgogna** [comment]
LGTM! :rocket:

**Comment by ericthewry** [comment]
LGTM

File: app/controllers/MathSwipeController.coffee (hunk 1 above)
Comment (bmwwoo) [inline app/controllers/MathSwipeController.coffee:101]: It tests it

**Pull request merged**